src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+
+    public String description;
+    public boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    
```

Comment:
Just an advice, this might not be a coding standard violation, but I think it is better if you make Task class an abstract class.

In addition, it is better to change the access modifier of variable description and isDone (on Line 3 and 4) to protected (following encapsulation fundamental, it is not advisable to declare your variable public)